Dynatrace/dynatrace-operator
# Adds EdgeConnect controller

## Description

Adds a controller for the new EdgeConnect CRD and an accompanying basic E2E test.

## How can this be tested?

* Deploy the operator
* create a simple EdgeConnect CR

```
apiVersion: dynatrace.com/v1alpha1
kind: EdgeConnect
metadata:
  name: myEdgeConnect
  namespace: dynatrace
spec:
  apiServer: "<tenant>.dynatrace.com"

  oauth:
    clientSecret: "secret"
    endpoint: "endpoint"
    resource: "urn"
```

* deploy this EdgeConnect CR
* check operator logs for "reconcling edgeconnct" log entries
* check that the "UpdatedTimestamp" filed in the EC status gets updated
* run E2E test with : `make test/e2e/edgeconnect`

## Checklist
